lib/blemulator_core.dart

```diff
@@ -1,11 +1,13 @@
 part of blemulator;
 
+/// Entry point for using the simulator
 class Blemulator {
 
   static Blemulator _instance = Blemulator._internal();
 
   DartToPlatformBridge _toPlatformBridge;
   SimulationManager _simulationManager;
+  // ignore: unused_field
```

Comment:
Why can't we remove it then? It will not be hard in the future to add it again.

Reply:
Discussed in person.
tl;dr: constructor of PlatformToDartBridge registers itself as a method call handler on a method channel and it would look weird to just call a constructor without any assignment of the result object to a field.